lib/sisimai/bite/email/verizon.rb

```diff
@@ -0,0 +1,265 @@
+module Sisimai::Bite::Email
+  module Verizon
+    # Sisimai::Bite::Email::Verizon parses a bounce email which created by
+    # Verizon Wireless.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Verizon.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :'received'  => %r/by .+[.]vtext[.]com /,
+        :'vtext.com' => {
+          :'from' => %r/\Apost_master[@]vtext[.]com\z/,
+        },
+        :'vzwpix.com' => {
+          :'from'    => %r/[<]?sysadmin[@].+[.]vzwpix[.]com[>]?\z/,
+          :'subject' => %r/Undeliverable Message/,
+        },
+      }
+      Indicators = Sisimai::Bite::Email.INDICATORS
+
+      def description; return 'Verizon Wireless: http://www.verizonwireless.com'; end
+      def smtpagent;   return Sisimai::Bite.smtpagent(self); end
```

Comment:
__[rubocop]__
* Layout/EmptyLineBetweenDefs: Use empty lines between method definitions. [:ref](https://github.com/bbatsov/ruby-style-guide#empty-lines-between-methods)
* Style/SingleLineMethods: Avoid single-line method definitions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-single-line-methods)
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->